apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,133 @@
+<!-- JavaScript code -->
+<script>
+    // Your Svelte script here
+</script>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <script src="https://kit.fontawesome.com/c850f0d795.js" crossorigin="anonymous"></script>
+</head>
+
+<!-- Landing Page - Card -->
+<div class="flex items-center justify-center h-screen">
+
+    <!-- Landing Page - Content -->
+    <div class="w-11/12 h-screen p-10 border border-secondary rounded-md shadow-lg">
```

Comment:
Do not use fractions such as `w-11/12`

Reply:
fixed `with w-full max-w-7xl` and removed `border-secondary`